lib/task_helpers/exports.rb

```diff
@@ -0,0 +1,41 @@
+module TaskHelpers
+  class Exports
+    # Illegal characters: '/', '|'
+    REGEX_KEEP_SPACES   = %r{[/|]}
+
+    # Illegal characters: '/', '|', ' '
+    REGEX_REMOVE_SPACES = %r{[/| ]}
+
+    # Replacement character
+    REPLACEMENT         = '_'.freeze
+
+    # Replace characters in \p str that are not allowed in filenames.
+    # Illegal characters are matched from the regexes REGEX_REMOVE_SPACES
+    # (if \p keep_spaces = false ) REGEX_KEEP_SPACES (if \p keep_spaces = true).
+    # Illegal characters are replaced with @@replacement.
+    # @param [String] str string to replace
+    # @param [Hash] options If options[:keep_spaces] == true, then keep spaces in
+    # \p str , otherwise replace spaces. (default = false, replace spaces)
+    def self.replace_chars(str, options = {})
```

Comment:
I feel like if this had a better name we wouldn't need the comment.

Maybe something like `safe_filename`?

Also it would document itself better if we passed the option specifically rather than passing the whole options hash for one value. Then the method would look like `def self.safe_filename(filename, keep_spaces = false)` which is a lot more clear.